core/lib/resources/presence/presence_manager.js

```diff
@@ -216,13 +201,11 @@ PresenceManager.prototype._implicitDisconnect = function(socketId, userId,
 
 PresenceManager.prototype.isOnline = function(message) {
   // Return with online, or with sentry, or with unexpired legacy msg
-  return (message.online && (message.sentry ||
-       (message.at && message.at >= (Date.now() - PresenceManager.autoPubTimeout))));
+  return (message.online && message.sentry);
 };
 
 PresenceManager.prototype.isExpired = function(message) {
-  return (message.online && !message.sentry &&
-      message.at && message.at < (Date.now() - PresenceManager.autoPubTimeout));
+  return (message.online && !message.sentry);
```

Comment:
section https://github.com/zendesk/radar/pull/149/files#diff-f6e0f89bf9a35e4181b67b5526da78fcR239 is meaningless,  and should be cleaned up